### ISSUE TITLE
Not working on windows

### ISSUE THREAD

**Issue by Meshredded** [fusesource/jansi#161]
The following code :
```
AnsiConsole.systemInstall();
System.out.println( ansi().eraseScreen().fg(GREEN).a("✓ Compilation Completed Successfully").reset() );
AnsiConsole.systemUninstall();
```

Renders :

`←[32m? Compilation Completed Successfully←[0m`

On windows 7 x64.

**Comment by frossm** [comment]
I am having a similar problem on windows 10.

**Comment by frossm** [comment]
With my build of Windows 10 x64 (18362) I was able to get get JANSI to work with the native cmd.exe ANSI processor.  I had to create the following key in the registry and assign it a DWORD value of 1

`HKCU\Console\VirtualTerminalLevel`

I restarted cmd.exe and the colors now appear correctly.  I'm sure this won't work for Windows 7 as it enables the windows 10 ANSI capabilities, but thought I would include it here should you upgrade or if others need it.

Michael
